cli/install/install.go

```diff
@@ -304,10 +304,10 @@ func programDependenciesInstalled(program string) bool {
 				" are installed.\n"+
 				"Proceed without checking?")
 			if !answer || err != nil {
-				return false
+				return false, err
```

Comment:
It could be reasonable to return an error status code in the case too:

```
$ sudo pacman -S firefox
resolving dependencies...
looking for conflicting packages...

Packages (1) firefox-127.0.1-1

Total Download Size:    69.04 MiB
Total Installed Size:  240.98 MiB
Net Upgrade Size:        1.45 MiB

:: Proceed with installation? [Y/n] n
$ echo $?
1
```